tests/admin/test-class-boldgrid-backup-admin-notices.php

```diff
@@ -28,25 +28,25 @@ public function test_notice_not_displayed_first_install() {
 		$this->assertTrue( $this->notice_counts_in_nav( '1.16.16', '1.16.16', 0 ) );
 	}
 
-	public function test_one_notice_displayed() {
-		$this->assertTrue( $this->notice_counts_in_nav( '1.12.16', '1.16.16', 1 ) );
-	}
+	// public function test_one_notice_displayed() {
```

Comment:
Was all this code meant to be commented out?

Reply:
Actually.....It was meant to be removed.....and now it has.